packages/flutter_form_bloc/lib/src/dropdown_field_bloc_builder.dart

```diff
@@ -264,16 +264,21 @@ class _DropdownFieldBlocBuilderState<Value>
 
   void _onDropdownPressed() async {
     if (widget.selectFieldBloc.state.items.isNotEmpty) {
-//TODO: Trick: https://github.com/flutter/flutter/issues/18672#issuecomment-426522889
-      if (_keyboardVisibility.isKeyboardVisible) {
+      if (_keyboardUtils.isKeyboardOpen) {
         _effectiveFocusNode.requestFocus();
-        await Future<void>.delayed(Duration(milliseconds: 1));
-        _effectiveFocusNode.unfocus();
-        await Future<void>.delayed(Duration(
-            milliseconds:
-                widget.millisecondsForShowDropdownItemsWhenKeyboardIsOpen));
+        await Future<void>.delayed(Duration(milliseconds: 10));
```

Comment:
If keyboard is open please use
```dart
await Future<void>.delayed(Duration( milliseconds: widget.millisecondsForShowDropdownItemsWhenKeyboardIsOpen));
```
This is for solve other bug of flutter dropdown, in case the layout changes. https://github.com/flutter/flutter/issues/22075

Reply:
Done ;)